### ISSUE TITLE
Get Iris data into usable format for neural network

### ISSUE THREAD

**Issue by ANGELLOPARR** [ANGELLOPARR/csc466-project#7]
We need the data from the UCI Machine Learning repository for Iris flowers in a format that can be used by our program. This data can be found [here](https://archive.ics.uci.edu/ml/datasets/iris). Pandas is probably the best option for this. What's important is that the output should be formatted in a multi-class format (one-hot encoding).

Definition of done: Pandas DataFrame (or equally good ML-friendly format) with training data and target data for the Iris Dataset. The target data should be N columns, 1 for each possible label and a 0 in all columns except the column that is the type of flower.
